stories/Modal/Modal.story.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+
+import Modal from '../../components/Modal';
+
+import { Header } from '../shared';
+
+import { 
+  Default,
+  WithClosingDisabled
+} from './Modal.stories.jsx';
+
+<Meta title="Modal" component={Modal} />
+
+<Header title="Modal">
+   Modals inform users about a task and can contain critical information,
+      require decisions, or involve multiple tasks.
+</Header>
+
+- [Importing and usage](#importing)
+- [API](#api)
+- [Examples](#examples)
+
+## <a id="importing"></a>Importing and usage
+
+```js
+import { Modal, Button } from '@catho/quantum';
+```
+
+<Canvas>
+  <Story story={Default} />
+</Canvas>
+
+## <a id="api"></a>API
+
+<ArgsTable of={Modal} />
+
+## <a id="examples"></a>Examples
+
+
+### With Closing Disabled
+
+```jsx
+...
+import { Modal, Button } from '@catho/quantum';
+
+export default function ModalWithClosingDisabled() {
+  const [openModal, setOpenModal] = useState(false);
+
+  return (
+    <>
+      <Button onClick={() => setOpenModal(true)}>Click to open Modal</Button>
```

Comment:
This button has the text "Click to open Modal", but in rendering the text is just "Open Modal". I find it necessary to keep them exactly the same